gemrb/core/Interface.cpp

```diff
@@ -2157,20 +2163,17 @@ bool Interface::InitializeVarsWithINI(const path_t& iniFileName)
 	PluginHolder<DataFileMgr> ini = MakePluginHolder<DataFileMgr>(IE_INI_CLASS_ID);
 	FileStream* iniStream = FileStream::OpenFile(iniFileName);
 	// if filename is not set we assume we are creating defaults without an INI
-	bool opened = ini->Open(iniStream);
+	bool opened = iniStream != nullptr && ini->Open(std::unique_ptr<DataStream>{iniStream});
```

Comment:
these two checks seem redundant. If they aren't then that implies `Open` cannot handle a nullptr, which it probably should.